en/peripherals/gripper.md

```diff
@@ -0,0 +1,106 @@
+# Gripper Peripherals
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
+Grippers are a commonly used mechanism for UAV delivery applications. There are multiple types of interfaces (e.g. PWM, CAN) for different products in the market.
+
+## Supported Gripper Types
+
+PX4 supports the following Gripper types, check the individual page to learn how to integrate them.
+
+- [Servo Gripper with PWM](gripper_servo.md)
+
+:::note
+The [Roboclaw driver](../modules/modules_driver.md#roboclaw) isn't supported as a Gripper yet. However in the future it may get supported as a Gripper.
+:::
+
+## Gripper Actuator Mapping
```

Comment:
Should this go into the servo docs? I appreciate that you might later have a UAVCAN servo that is almost the same, but I think in that case it might still be good to have a separate image. Thoughts?

Reply:
Actually I think "yes", along with the specific configuration for the servo. There might be duplication, but the servo setup will be easier to follow.

Reply:
I moved most of this to the Gripper Servo (along with the image, as it is PWM specific), and left generic comments only in the gripper doc